conversations/push-ios/didreceiveremote/didreceiveremote.swift

```diff
@@ -0,0 +1,18 @@
+// For iOS 10 and later; do not forget to set up a delegate for UNUserNotificationCenter
+func userNotificationCenter(_ center: UNUserNotificationCenter,
+            didReceive response: UNNotificationResponse,
```

Comment:
I don't know whether we have our own standards at Twilio for styling Swift code, but it would seem that line continuations such as this should actually only be indented by 2 (or 4 here) spaces, rather than trying to line them up with something particular in the previous line or whatever is going on here.

Reply:
They're supposed to be lined up with the parameters vertically aligned (according to swiftlint). I did reformat this one with swiftlint, but couldn't get it to pass the linting rules, so I copied exactly how Apple formatted it.